gen/calc.py

```diff
@@ -622,13 +622,11 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
             'rexray': {
                 'loglevel': 'info',
                 'modules': {
-                    'default-admin': {
```

Comment:
This is correct. The administrative module was around in REX prior to libStorage. The module was intended to be place that would have its own administrative API that gave you access to all of the REX CLI activity. The direction instead was to create the lower layer API (libStorage API) to provide similar types of access.